packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -61,6 +64,17 @@ const DMT5TradeModal = observer(
             return undefined;
         };
 
+        const mobileURLSet = () => {
+            setURL(window.location.replace(deepLink) as unknown as string) as unknown as string;
```

Comment:
this type assertion is a bit confusing to me...

Reply:
That's the way to avoid the typescript error since we're assigning window.location value to string type.

Reply:
@hirad-deriv I understand the urgency of the task but can you add a comment here to fix the issue with the type.

Reply:
Sure